pkg/controller/operators/olm/operator.go

```diff
@@ -1824,6 +1818,12 @@ func (a *Operator) updateInstallStatus(csv *v1alpha1.ClusterServiceVersion, inst
 		return strategyErr
 	}
 
+	if condErr != nil {
+		a.logger.WithError(condErr).Debug("operator is not upgradeable")
+		csv.SetPhaseWithEventIfChanged(v1alpha1.CSVPhasePending, v1alpha1.CSVReasonOperatorConditionNotUpgradeable, "operator upgradeable condition is false", now, a.recorder)
```

Comment:
I don't think we want to flip back to pending - that will attempt to recreate / reinstall components. Shouldn't it stay succeeded, just not upgrade?

Reply:
I agree with @ecordell